### PR TITLE
SpecAugment: max_num_masks is at least min_num_masks

### PR DESCRIPTION
**Pull request by JackTemaki**
I triggered one "last batch" in Tedlium that had only one sequence that was apparently so short that max_masks got smaller than min_masks for the time.

### CODE REVIEW

**Review by kuacakuaca — APPROVED** [review]

File: i6_models/primitives/specaugment.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import numpy as np
```
Comment (albertz) [inline i6_models/primitives/specaugment.py:1]: We mostly use `import numpy` in other code (consistent to `import torch`), here also in the repo (although not too much yet), but also in RETURNN.
Although this variant here is more the official recommended way.

So maybe for i6_models, we can independently decide now what variant to pick and then always keep it consistent.

Comment (albertz) [reply i6_models/primitives/specaugment.py:1]: You merged without resolving this here, or commenting here. So what do you mean?

Comment (albertz) [reply i6_models/primitives/specaugment.py:1]: Ah now I see #38.

Comment (JackTemaki) [reply i6_models/primitives/specaugment.py:1]: As this is an independent non-critical issue, I would not delay this PR by that. If we decide different to this PR we can just change it back.

**Pull request merged**